api/app/config/datafeeds.json

```
@@ -14,7 +14,7 @@
         },
         {
             "name": "GLBRC",
-            "url": "https://fair-data.glbrc.org/glbrc.json"
+            "url": "https://fair-data.glbrc.org/glbrc-0.1.0.json"
```

I'd like to revert this to the original url and use the versioned form only for testing.
```suggestion
            "url": "https://fair-data.glbrc.org/glbrc.json"
```